src/atomizer.js

```diff
@@ -295,10 +295,22 @@ Atomizer.prototype.parseConfig = function (config/*:AtomizerConfig*/, options/*:
                     value = `var(${matchVal.groups.cssVariable})`;
                 }
                 if (matchVal.groups.named) {
-                    // first check if 'inh' is the value
+                    // add global values first
                     if (matchVal.groups.named === 'inh') {
                         value = 'inherit';
                     }
+                    else if (matchVal.groups.named === 'ini') {
```

Comment:
I guess the one risk with this is that someone is using a custom value called `ini` or `un` or etc, and that would be a breaking change